backend/tarsy/main.py

```
@@ -417,44 +402,19 @@ def deep_sanitize(obj):
-    except WebSocketDisconnect:
-        pass
-    except Exception as e:
-        logger.error(f"WebSocket error for alert {alert_id}: {str(e)}")
-    finally:
-        websocket_manager.disconnect(websocket, alert_id)
 
+@app.get("/session-id/{alert_id}")
+async def get_session_id(alert_id: str):
+    """Get session ID for an alert.
+    Needed for dashboard websocket subscription because
+    the client which sent the alert request needs to know the session ID (generated later)
```

Ruff (0.12.2)

Line too long (90 > 88)

(E501)